### PR TITLE
[EHR-775] Reorganize the repo structure for dashboard separation

### PR DESCRIPTION
**Pull request by xj2193**
- Add the NIH Award Metrics and general metrics dashboards into the repo
- Create folder structure under "sql" folder
- Move sql queries to corresponding dashboard folders

### CODE REVIEW

File: restore_workbook.py [hunk 1]
```diff
@@ -7,8 +7,11 @@
 from dotenv import dotenv_values
 
 CIPHER_FILE = '.cipher'
-WORKBOOKS = [Path(__file__).parent / 'EHR Ops Dashboard.twb',
-                Path(__file__).parent / 'Data_Transfer_Rate.twb']
+DASHBOARDS_PATH = Path(__file__).parent / 'dashboard_metrics'
```
Comment (jp3477) [inline restore_workbook.py:10]: Thanks for updating this!

File: strip_workbook.py [hunk 2]
```diff
@@ -45,7 +49,7 @@ def reverse_dict(d):
 
 
 def contains_dict(s, d):
-    for key, value in d.items():
+    for key, _value in d.items():
```
Comment (jp3477) [inline strip_workbook.py:52]: This works, but curious why you added the underscore.

File: dashboard_metrics/EHR Ops General Data Quality Dashboard/sql/ehr_metrics/general_metrics/v_org_hpo_mapping.sql [hunk 3]
```diff
@@ -0,0 +1,34 @@
+-- This query returns the org_id linkage between CDR and PDR
+WITH
```
Comment (jp3477) [inline dashboard_metrics/EHR Ops General Data Quality Dashboard/sql/ehr_metrics/general_metrics/v_org_hpo_mapping.sql:2]: Should we remove the v_ from the query name? Just thinking about appending the v_ to each file name when generating the views.

File: app_engine_std/cron/app/jobs/refresh_snapshot_tables.py [hunk 4]
```diff
@@ -30,16 +30,14 @@ def run(self):
...
-            'dc_3',
-            'dc_4'
+           'mv_patient_status'
         ]
 
         for table in snapshot_tables:
             payload = {
-                'dataset': 'dev_testing',
-                'table': table
+                'dataset': 'ehr_ops_metrics_staging',
```
Comment (jp3477) [inline app_engine_std/cron/app/jobs/refresh_snapshot_tables.py:38]: Eventually I think we can try to store these dataset names in variables somewhere. But for now, hardcoding it is great.

File: app_engine_std/cron/app/jobs/refresh_snapshot_tables.py [hunk 5]
```diff
@@ -30,16 +30,14 @@ def run(self):
         _logger.info('Creating Cloud Tasks to refresh snapshot tables')
 
         snapshot_tables = [
-            'dc_1',
-            'dc_2',
-            'dc_3',
-            'dc_4'
+           'mv_patient_status'
```
Comment (jp3477) [inline app_engine_std/cron/app/jobs/refresh_snapshot_tables.py:33]: We'll add the rest here on this PR or another.

**Review by jp3477 — APPROVED** [review]
Hey Zoey, this looks great. Approved with some light commentary.

File: dashboard_metrics/EHR Ops General Data Quality Dashboard/sql/ehr_metrics/general_metrics/v_org_hpo_mapping.sql (hunk 3 above)
Comment (xj2193) [reply dashboard_metrics/EHR Ops General Data Quality Dashboard/sql/ehr_metrics/general_metrics/v_org_hpo_mapping.sql:2]: Yes we should. This one is the script to generate the view in operations_analytics. We don't need to recreate the view in `aou_curation_ehr_ops` project.

File: app_engine_std/cron/app/jobs/refresh_snapshot_tables.py (hunk 4 above)
Comment (xj2193) [reply app_engine_std/cron/app/jobs/refresh_snapshot_tables.py:38]: Sounds great

**Pull request merged**